internal/guest/storage/pmem/pmem.go

```diff
@@ -21,9 +21,13 @@ import (
 
 // Test dependencies
 var (
-	osMkdirAll  = os.MkdirAll
-	osRemoveAll = os.RemoveAll
-	unixMount   = unix.Mount
+	osMkdirAll         = os.MkdirAll
+	osRemoveAll        = os.RemoveAll
+	unixMount          = unix.Mount
+	mountInternal      = mount
+	createLinearTarget = dm.CreateZeroSectorLinearTarget
+	veritySetup        = dm.CreateVerityTarget
+	removeDevice       = dm.RemoveDevice
```

Comment:
Should keep the tradition of naming the mocked out aliases to the same name as the original function but different casing. 
veritySetup -> createVerityTarget
createLinearTarget -> createZeroSectorLinearTarget